### ISSUE TITLE
Using DecontX

### ISSUE THREAD

**Issue by barbarashih** [campbio/celda#367]
I am wondering if the count matrix returned from DecontX can be used for downstream analysis like stats comparison and visualisation with FeaturePlot in Seurat?

**Comment by joshua-d-campbell** [comment]
Yes it can! Generally, the corrected count matrix can replace the original count matrix as the primary input into Seurat. We are in the process of updating our vignette to show how to move counts back and forth between Seurat and a SingleCellExperiment object. For now, here is a brief snippet of code which you can use to get decontX counts into a Seurat object:

```
# Read counts from CellRanger output
library(Seurat)
counts <- Read10X("sample/outs/filtered_feature_bc_matrix/")

# Create a SingleCellExperiment object and run decontX
sce <- SingleCellExperiment(list(counts = counts))
sce <- decontX(sce)

# Create a Seurat object from a SCE with decontX results
seuratObject <- CreateSeuratObject(round(decontXcounts(sce)))
```

Optionally, the "raw" matrix can be also be imported and used as the background:

```
counts.raw <- Read10X("sample/outs/raw_feature_bc_matrix/")
sce.raw <- SingleCellExperiment(list(counts = counts.raw))
sce <- decontX(sce, background = sce.raw)
```

Note that the decontaminated matrix of decontX consists of floating point numbers and must be rounded to integers before adding it to a Seurat object. If you already have a Seurat object containing the counts matrix and would like to run decontX, you can retrieve the count matrix, create a SCE object, and run decontX, and then add it back to the Seurat object:

```
counts <- GetAssayData(object = seuratObject, slot = "counts")
sce <- SingleCellExperiment(list(counts = counts))
sce <- decontX(sce)
seuratObj[["decontXcounts"]] <- CreateAssayObject(counts = decontXcounts(sce))
```


I am going to move this into the Discussions section rather than Issues and edit the title to say "Using DecontX with Seurat" so others will be able to see easily find it.
